conf/upf.json

```
@@ -61,12 +63,14 @@
 
     "": "Gateway interfaces",
     "access": {
-        "ifname": "ens803f2"
+        "ifname": "ens803f2",
+        "cndp_jsonc_file": "/opt/bess/bessctl/conf/cndp_upf_1worker.jsonc"
```

Given that DPDK is the default option, we should leave/have the `cndp_jsonc_file` as a comment
```suggestion
        "": "cndp_jsonc_file: /opt/bess/bessctl/conf/cndp_upf_1worker.jsonc"
```

---

Keeping this uncommented won't affect DPDK mode. So one less thing to manually change when using DPDK or CNDP mode. But if you still prefer to keep it commented I can do.

---

I am/was thinking from the end user perspective (especially for new users) that it is better to have `cndp_jsonc_file` as "comment", so they know what is involved when using the UPF in CNDP mode. What do you think?